src/main/java/com/github/webicitybrowser/webicitybrowser/engine/protocol/WebicityProtocol.java

```diff
@@ -0,0 +1,49 @@
+package com.github.webicitybrowser.webicitybrowser.engine.protocol;
+
+import com.github.webicitybrowser.spec.url.URL;
+import com.github.webicitybrowser.webicity.core.net.Connection;
+import com.github.webicitybrowser.webicity.core.net.Protocol;
+import com.github.webicitybrowser.webicity.core.net.ProtocolContext;
+
+import java.io.*;
+
+public class WebicityProtocol implements Protocol {
+	@Override
+	public String[] getSchemes() {
+		return new String[]{"webicity"};
```

Comment:
Stylistic choice, but can you add spaces between `[]` and `{`, as well as between the brackets and string?